pom.client.xml

```diff
@@ -430,6 +508,10 @@
                 <title>Azure Event Hubs</title>
                 <packages>com.microsoft.azure.eventhubs*:com.microsoft.azure.eventprocessorhost</packages>
               </group>
+              <group>
+                <title>Azure Common</title>
+                <packages>com.microsoft.azure.common*</packages>
+              </group>
```

Comment:
I would specify separate groups for 'Azure Common', 'Azure Common - Management', and 'Azure Common - Authentication'.

Reply:
Done